osu.Game.Rulesets.Taiko/Edit/TaikoHitObjectComposer.cs

```diff
@@ -18,6 +19,16 @@ public TaikoHitObjectComposer(TaikoRuleset ruleset)
         {
         }
 
+        public override void EndPlacement(HitObject hitObject, bool commit)
+        {
+            EditorBeatmap.PlacementObject.Value = null;
+
+            if (commit)
+            {
+                EditorBeatmap.Add(hitObject);
+            }
+        }
```

Comment:
Should not be done only for taiko like you have it. Should be a toggle for all rulesets which would allow changing the behaviour whether to seek to the end of the object or not upon ending placement.

See also: https://github.com/ppy/osu/discussions/17424#discussioncomment-2426957

Highly suggest doing this in a different PR than this one.

Reply:
Yep, this PR should just fix the preview placement not showing and the remaining portion should 100% be a settings applied not at a ruleset level.

Reply:
ok, I reverted it

Reply:
Not everything has been reverted. There are code inspections left behind in this file now.

Reply:
I left a using(my mistake)
I will fix it in next push